extensions/ql-vscode/src/view/model-editor/MethodRow.tsx

```diff
@@ -199,54 +193,41 @@ const ModelableMethodRow = forwardRef<HTMLElement | undefined, MethodRowProps>(
             )}
           </>
         )}
-        {!props.modelingInProgress && (
-          <>
-            <MultiModelColumn>
-              {modeledMethods.map((modeledMethod, index) => (
+        {!props.modelingInProgress &&
+          modeledMethods.map((modeledMethod, index) => (
+            <>
+              <DataGridCell>
                 <ModelTypeDropdown
-                  key={index}
```

Comment:
I was a little concerned about removing these keys. I've done manual tested and I haven't seen anything not updating correctly. Also, the default behaviour if you don't specify a key is to use the index.

Reply:
Ah nice. I did try adding a key but it won't even allow it on a `<>`. Good suggestion to switch to `<Fragment>` instead.